frb/data/Galaxies/20190102C/FRB20190102C_host.json

```diff
@@ -125,7 +184,9 @@
         "WFC3_F160W_flux_err": 0.00013276959821990165,
         "WFC3_F160W_ref": "Mannings2021",
         "WFC3_F300X": 25.992214395230807,
-        "WFC3_F300X_flux": 25.992214395230807,
+        "WFC3_F300X_err": 999.0,
```

Comment:
please confirm this should be an upper limit

Reply:
Yep, that's a real upper limit